constellations/gemini-user/src/internship/internship.js

```diff
@@ -78,59 +78,52 @@ export class InternshipPage extends React.Component {
                     students who have done internships for both teams!
                 </p>
 
-                <table id="past-intern">
-                    <caption>Past Interns:</caption> <br></br>
-                    <tr>
-                        <th>Name</th>
-                        <th>School</th>
-                        <th>College</th>
-                    </tr>
-                    <tr>
-                        <td>Cathy Y.</td>
-                        <td>Montgomery Blair HS 2020</td>
-                        <td>Massachusetts Institute of Technology (MIT)</td>
-                    </tr>
-                    <tr>
-                        <td>Jessica Y.</td>
-                        <td>Montgomery Blair HS 2020</td>
-                        <td>University of Maryland</td>
-                    </tr>
-                    <tr>
-                        <td>Max Z.</td>
-                        <td>Montgomery Blair HS 2021</td>
-                        <td> </td>
-                    </tr>
-                    <tr>
-                        <td>Chujia G.</td>
-                        <td>Montgomery Blair HS 2022</td>
-                        <td> </td>
-                    </tr>
-                    <tr>
-                        <td>Frederick Z.</td>
-                        <td>Montgomery Blair HS 2023</td>
-                        <td> </td>
-                    </tr>
-                    <tr>
-                        <td>Tony W.</td>
-                        <td>Richard Montgomery HS 2021</td>
-                        <td> </td>
-                    </tr>
-                    <tr>
-                        <td>Austin H.</td>
-                        <td>Winston Churchill HS 2017</td>
-                        <td>Carnegie Mellon University</td>
-                    </tr>
-                    <tr>
-                        <td>Serena X.</td>
-                        <td>Winston Churchill HS 2020</td>
-                        <td>University of Pennsylvania</td>
-                    </tr>
-                    <tr>
-                        <td>Daniel L.</td>
-                        <td>Winston Churchill HS 2021</td>
-                        <td> </td>
-                    </tr>
-                </table>
+                <h4>Past Interns:</h4>
+                <div class="intern-main">
+                    <p class="name">Cathy Y.</p>{" "}
```

Comment:
what does `{" "}` do here? when i tested it they didn't seem to make a difference in the page

Reply:
not too sure. prettier added it